tests/test_batch_renorm.py

```
@@ -0,0 +1,155 @@
+        gamma, beta, mean, var = get_params(3)
+
+        with marquetry.test_mode():
+            y = funcs.batch_renormalization(x, gamma, beta, mean.copy(), var.copy(), rmax=3.0, dmax=5.0)
+
+        expected = gamma * (x - mean) / np.sqrt(var + eps) + beta
+        self.assertTrue(array_close(y.data, expected))
+
+    def test_backward_default(self):
+        x = np.random.randn(8, 3)
+        gamma, beta, mean, var = get_params(3)
+        f = lambda x_in: funcs.batch_renormalization(x_in, gamma, beta, mean.copy(), var.copy())
```

Ruff (0.15.15)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `f` as a `def`

(E731)